dropshot/tests/test_config.rs

```diff
@@ -74,15 +171,15 @@ async fn test_config_bind_address() {
         path: log_path.clone(),
         if_exists: dropshot::ConfigLoggingIfExists::Append,
     };
-    let log = log_config.to_logger("test_config_bind_address").unwrap();
+    let log = log_config.to_logger("test_config_bind_address_http").unwrap();
 
     let client = hyper::Client::new();
     let bind_ip_str = "127.0.0.1";
-    let bind_port: u16 = 12215;
+    let bind_port = 12215;
 
     /*
      * This helper constructs a GET HTTP request to
-     * http://$bind_ip_str:$port/, where $port is the argument to the
+     * https://$bind_ip_str:$port/, where $port is the argument to the
```

Comment:
Should this be `http`?

Reply:
Yes, thank you. I apparently updated the wrong comment